netbox/virtualization/forms/model_forms.py

```diff
@@ -343,20 +343,26 @@ class VMInterfaceForm(InterfaceCommonForm, VMComponentForm):
         required=False,
         label=_('VRF')
     )
+    vlan_translation_policy = DynamicModelChoiceField(
+        queryset=VLANTranslationPolicy.objects.all(),
+        required=False,
+        label=_('VLAN Translation Policy')
+    )
 
     fieldsets = (
         FieldSet('virtual_machine', 'name', 'description', 'tags', name=_('Interface')),
         FieldSet('vrf', 'mac_address', name=_('Addressing')),
         FieldSet('mtu', 'enabled', name=_('Operation')),
         FieldSet('parent', 'bridge', name=_('Related Interfaces')),
         FieldSet('mode', 'vlan_group', 'untagged_vlan', 'tagged_vlans', name=_('802.1Q Switching')),
+        FieldSet('vlan_translation_policy', name=_('VLAN Translation'))
```

Comment:
As with dcim.InterfaceForm, this can be merged into the "802.1Q switching" field set.